spec/services/github_service_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
```

Comment:
Thanks for making this test dynamic, I can add the feature where we can get GitHub users who are following  the logged in user.